crates/sargon_SPLIT_ME/tests/integration/main.rs

```diff
@@ -490,5 +843,709 @@ mod integration_tests {
                 )
             );
         }
+
+        mod multi_tx {
```

Comment:
migrated over unit tests from `signing` module, which does NOT depend on profile (and we dont want it too!) so those unit tests in `signing` crate (new) which were referencing Profile have been moved into an integration test here in Sargon crate. (I had to make some pub(crate) symbols -> `pub` and remove `cfg(test)`. but that is fine!).